pysite/migrations/tables/users/v1.py

```diff
@@ -0,0 +1,7 @@
+def run(db, table, table_obj):
+    for document in db.get_all(table):
+        if "email" in document:
+            del document["email"]
+
+            db.insert(table, document, conflict="update", durability="soft")
+    db.sync(table)
```

Comment:
I feel like these tests should have some docs to them. I can *somewhat* get behind that this is removing all email fields from users, but I feel like having a docstring describe it would be nice regardless. or alternatively, the django way:tm:, have the migration filenames reflect it, e.g. `v1_remove_email_field`. not sure if the migration engine supports that though

Reply:
Done.